components/log-viewer-webui/client/src/api/query.js

```diff
@@ -13,20 +13,21 @@ import axios from "axios";
  */
 
 /**
- * Submits a job to extract the split of an original file that contains a given log event. The file
- * is extracted as a CLP IR file.
+ * Submits a job to extract the stream that contains a given log event. The stream is extracted
+ * either as a CLP IR or a JSON line file.
  *
- * @param {number|string} origFileId The ID of the original file
+ * @param {number} extractJobType Type of the extraction job
+ * @param {number|string} streamId The ID of the original stream
```

Comment:
I could be wrong but could this parameter always be of type `string`?

Reply:
you are right. thanks for catching and fixed